about.html

```diff
@@ -105,17 +107,17 @@ <h2 property="schema:name">Features</h2>
                   <p><strong>Organizations Deploying Solid</strong></p>
                   <p>See organizations working to support the deployment of Solid technology across a range of
                   sectors.</p>
-                  <a href="" rel="rdfs:seeAlso">See organizations</a>
+                  <a href="/for-organizations.html" rel="rdfs:seeAlso">See organizations</a>
                 </li>
                 <li>
                   <p><strong>Develop a Solid application</strong></p>
                   <p>Find tutorials, repositories, libraries, and vocabularies.</p>
-                  <a href="" rel="rdfs:seeAlso">Develop an application</a>
+                  <a href="/for-developers.html" rel="rdfs:seeAlso">Develop an application</a>
                 </li>
                 <li>
                   <p><strong>Example applications</strong></p>
                   <p>Find examples of applications built with Solid.</p>
-                  <a href="" rel="rdfs:seeAlso">See example applications</a>
+                  <a href="https://forum.solidproject.org/c/applications/" rel="rdfs:seeAlso">See example applications</a>
```

Comment:
```suggestion
                  <a href="https://solidproject.org/apps" rel="rdfs:seeAlso">See example applications</a>
```

What's currently on the website is most comprehensive and up to date. Any particular reason to direct people to the forum?

Reply:
This was not part of my update (I only removed the class button).

I agree that https://solidproject.org/apps is (probably) better, but then we should add that page to the new redesign as well.

Maybe @KyraAssaad have some thoughts on this?